### ISSUE TITLE
Link to talk

### ISSUE THREAD

**Issue by mark-dropbear** [johnpryan/architecture_talk#1]
Hi John,

Just wanted to ask if this was for a previous or future talk and if it already exists if there was a link I could watch it at?

Regards,
Mark

**Comment by johnpryan** [comment]
No, sorry this was for a private event, but I may give the talk again in the future. Thanks for your interest!
